onnxruntime/core/providers/snpe/snpe.h

```diff
@@ -0,0 +1,130 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include <sstream>
+#include <iostream>
+#include <memory>
+#include <unordered_map>
+#include <string>
+#include <vector>
+#include "core/common/common.h"
+#include "core/framework/op_kernel.h"
+#include "core/framework/tensor.h"
+#include "core/providers/snpe/snpe_lib.h"
+#include "core/providers/snpe/snpe_execution_provider.h"
+
+namespace onnxruntime {
+namespace contrib {
+namespace snpe {
+
+class SnpeKernel : public OpKernel {
+ public:
+  explicit SnpeKernel(const OpKernelInfo& info) : OpKernel(info) {
+    input_cout_ = info.GetInputCount();
```

Comment:
nit: could they be renamed to `input_count_` and `output_count_`?

Reply:
it's typo. :)